src/main/java/cstwooneohthree/glados/Glados.java

```diff
@@ -0,0 +1,135 @@
+package cstwooneohthree.glados;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import cstwooneohthree.glados.enums.TaskType;
+import cstwooneohthree.glados.enums.UiType;
+import cstwooneohthree.glados.exceptions.CommandNotFoundException;
+import cstwooneohthree.glados.exceptions.GladosException;
+import cstwooneohthree.glados.exceptions.TaskNotFoundException;
+import cstwooneohthree.glados.tasks.Task;
+import cstwooneohthree.glados.tasks.TaskList;
+import cstwooneohthree.glados.utils.Parser;
+import cstwooneohthree.glados.utils.Ui;
+
+/**
+ * Glados class to initiate and run CLI.
+ *
+ * @author jayjay19630
+ */
+public class Glados {
+    /* TaskList object to manage tasks */
+    private TaskList taskList;
+    private Ui ui;
+
+    /**
+     * Constructs Glados object by initializing TaskList.
+     * TaskList object is created with loading tasks.
+     */
+    public Glados(UiType uiType) {
+        this.taskList = new TaskList(true);
+        this.ui = new Ui(uiType);
+    }
+
+    private void runCommandLineInterface() {
+        // Greet the user
+        greet();
+
+        // Initialise a scanner for detecting input
+        Scanner sc = new Scanner(System.in);
+
+        // Run a loop indefinitely to execute commands until program exits through bye
+        while (true) {
+            String input = sc.nextLine().toLowerCase();
```

Comment:
I liked that you handled user input, ensuring the program works with case-insensitive command.